test/views/UploadLetterOfAuthorityViewSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import models.DraftId
+import models.upscan.UpscanInitiateResponse
+import play.twirl.api.HtmlFormat
+import views.behaviours.ViewBehaviours
+import views.html.UploadLetterOfAuthorityView
+
+class UploadLetterOfAuthorityViewSpec extends ViewBehaviours {
+
+  val messageKeyPrefix = "uploadLetterOfAuthority"
+
+  val upscanInitiateResponse: UpscanInitiateResponse = UpscanInitiateResponse(
```

Comment:
Maybe move this to the SpecBase as it is used by more than one view.

Reply:
Was going to make a constant file for these as noticed a lot of repetition for these in other tests too but maybe not in the PR, will move there for now.

Reply:
Another reason why I suggested this is because it was done for `testApplicationSummary` so leaving it in the SpecBase will suffice for now as that was moved to the SpecBase in this PR.

Reply:
Would also require less work when that is done in future.